internal/characters/cyno/cyno.go

```diff
@@ -20,6 +20,7 @@ type char struct {
 	c4Counter      int
 	c6Stacks       int
 	a1Extended     bool
+	normalBCounter int
```

Comment:
Don't want to change this because this changes visibility in golang. Until this is used outside of cyno's package, this should stay lowercase. I think lowercase `normalBHitNum` is more consistent since `normalHitNum` is in every file.

Reply:
Ok, use the lower case version then.